material-ui/material-ui-tests.tsx

```diff
@@ -223,7 +223,7 @@ const styles = {
     paddingTop: 16,
     marginBottom: 12,
     fontWeight: 400,
-  },
+  } as React.CSSProperties,
```

Comment:
It's a cast and yes it is a breaking change.

Check this:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a9657ab273092b706b22d6e0bb9dcbca40998248/material-ui/material-ui-tests.tsx#L170
and this: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a9657ab273092b706b22d6e0bb9dcbca40998248/react/index.d.ts#L1267

When writing `root: { justifyContent: 'space-around' }`, TypeScript thinks that `justifyContent` is of type `string` instead of `'space-around'` (it's like TypeScript is not smart enough).
`justifyContent` is defined as `"flex-start" | "flex-end" | "center" | "space-between" | "space-around"` and not `string`.
Thus you need a cast.

Good thing is that the cast fails if you write something like `root: { justifyContent: 'foobar' }  as React.CSSProperties`.

This is why I've introduced file `react/test/cssProperties.tsx` to show, test and experiment around this.